lib/views/after_auth_screens/app_settings/app_settings_page.dart

```diff
@@ -302,11 +302,16 @@ class AppSettingsPage extends StatelessWidget {
                         successText: 'LogOut',
                         success: () async {
                           try {
-                            await model.logout();
+                            final bool isLogoutSuccessful =
+                                await model.logout();
+                            if (!isLogoutSuccessful) {
+                              throw Error(); //checks whether the logout was successful or not.
```

Comment:
Where is this error catched ?

Reply:
@literalEval there's a catch block at line 316. please have a look at it.